pkg/netdevice/netInfoProviders.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package netdevice
+
+import (
+	"github.com/golang/glog"
+
+	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
+
+	"github.com/intel/sriov-network-device-plugin/pkg/types"
```

Comment:
Depending on whether your series or mine (https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pull/280) is merged first. I will either have to change this import to NPWG's repo, or you'll have to change to NPWG's.

Reply:
This one will likely take longer to review, so no worries. Thanks for the heads up

Reply:
Rebased